tests/build/build.go

```diff
@@ -408,49 +405,13 @@ var _ = framework.BuildSuiteDescribe("Build service E2E tests", Label("build", "
 				Succeed(), fmt.Sprintf("timed out waiting for gitops content to be created for app %s in namespace %s: %+v", app.Name, app.Namespace, err),
 			)
 
-			customBundleConfigMap, err := f.CommonController.GetConfigMap(constants.BuildPipelinesConfigMapName, testNamespace)
```

Comment:
removed, because build pipelines configmap [is going away](https://github.com/redhat-appstudio/infra-deployments/pull/1094)